Sources/CodeEditTextView/Controller/STTextViewController.swift

```diff
@@ -331,11 +340,14 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
         }
     }
 
-    // MARK: Key Presses
+    // MARK: Selectors
 
-    /// Handles `keyDown` events in the `textView`
     override public func keyDown(with event: NSEvent) {
-        // TODO: - This should be uncessecary
+        // This should be uneccessary but if removed STTextView receives some `keydown`s twice.
```

Comment:
Should we contact Marcin about this for a fix?

Reply:
This isn't an STTextView problem. It's the same functionality given by  NSTextView and I'm leaving it as TODO while I look for a solution that works in case someone else has an idea.